_includes/key-dates-section.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+    <h2 class="key-dates-title">KEY DATES</h2>
+    <div class="key-dates-wrapper">
+        <div class="key-date col-md-4">
+            <h3 class="key-dates-date">{{page.date1}}</h3>
+            <p class="key-dates-desc">{{page.date1desc}}</p>
+        </div>
+        <div class="key-date col-md-4">
```

Comment:
Same comment as below. Can this block be abstracted out?

Reply:
Same response as below.